### PR TITLE
Fixed bug where meteorBinary setting was ignored

### PR DESCRIPTION
**Pull request by ffxsam**
`meteorBinary` in mup.json was ignored. This has been fixed.

### CODE REVIEW

**Comment by ffxsam** [comment]
Can anyone merge this?

**Comment by ffxsam** [comment]
Oops, it needs to supply a default value if meteorBinary is not supplied.

File: lib/actions.js [hunk 1]
```diff
@@ -111,9 +111,10 @@ Actions.prototype.deploy = function() {
   var appName = this.config.appName;
   var appPath = this.config.app;
   var enableUploadProgressBar = this.config.enableUploadProgressBar;
+  var meteorBinary = this.config.meteorBinary;
```
Comment (arunoda) [inline lib/actions.js:114]: Add a default value. But not here, in the `config.js`

**Comment by arunoda** [comment]
Actually, it was there. We are good to go.

**Comment by arunoda** [comment]
Published as `0.10.8`

**Pull request merged**